src/test/java/com/google/sps/data/DatastoreEntityToObjectConverterTest.java

```diff
@@ -0,0 +1,86 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Constants;
+import com.google.sps.data.DatastoreEntityToObjectConverter;
+import com.google.sps.data.DatastoreManager;
+import com.google.sps.data.Post;
+import com.google.sps.data.Representative;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+@RunWith(JUnit4.class)
+public final class DatastoreEntityToObjectConverterTest {
+    private final LocalServiceTestHelper helper =
+        new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+    private DatastoreService ds; 
+    private Representative donaldTrump; 
+    private Entity repEntity; 
+
+    @Before
+    public void setUp() {
+        helper.setUp();
+        ds = DatastoreServiceFactory.getDatastoreService();
+
+        long commentId = DatastoreManager.insertCommentInDatastore("Anonymous", "Nice dude"); 
+        List<Long> commentIds = new ArrayList<>(); 
+        commentIds.add(commentId); 
+        long commentIdQuestion = DatastoreManager.insertCommentInDatastore("Anonymous", "Why are you in office?");
+        long commentIdAnswer = DatastoreManager.insertCommentInDatastore("Donald Trump", "Because I want to be.");
+
+        Entity postEntity = new Entity(Constants.POST_ENTITY_TYPE); 
+        postEntity.setProperty(Constants.POST_QUESTION, commentIdQuestion); 
+        postEntity.setProperty(Constants.POST_ANSWER, commentIdAnswer); 
+        postEntity.setProperty(Constants.POST_REPLIES, commentIds); 
+        ds.put(postEntity); 
+        long postId = postEntity.getKey().getId(); 
+        List<Long> postIds = new ArrayList<>(); 
+        postIds.add(postId); 
+
+        this.repEntity = new Entity(Constants.REP_ENTITY_TYPE); 
```

Comment:
don't need this here. typically only use this in constructors to distinguish between params and instance vars. That shouldn't be a problem here.